src/Routing/Route/Route.php

```diff
@@ -199,9 +199,13 @@ protected function _writeRoute()
             $parsed = preg_replace('#/\\\\\*$#', '(?:/(?P<_args_>.*))?', $parsed);
             $this->_greedy = true;
         }
+        $mode = '';
+        if (!empty($this->options['multibytePattern'])) {
```

Comment:
Would be nice to have class constants for those. Magic string usage from route files is typo friendly.